### ISSUE TITLE
Release 1.3.0

### ISSUE THREAD

**Issue by Thrameos** [jpype-project/jpype#979]
- [x] Start from the release branch
      ``git checkout release``
- [x] Make a new branch for the release cycle
      ``git checkout -b releases/{version}``
- [x] Merge the current master with the release
      ``git pull origin master``
- [x] Start a release
      ``bumpversion release``
- [x] Edit doc/CHANGELOG.rst
- [x] Send the release to be evaluated
      ``git push``
- [x] Verify CI on azure
- [x] Manually trigger a ``jpype.release`` on azure
      If successful, download the artifacts for publication.
- [x] Advance the release pointer
      ``git checkout release``
      ``git merge releases/<version>``
- [x] Publish the release
  - Add draft release on github
  - Attach the artifacts to the release.
- [x] Start master on a new cycle
  - Use a PR to pull release back to master
  - ``git checkout master``
  - ``git checkout -b cycle``
  - ``git merge release``
  - ``bumpversion patch``
  - Use PR to insert the cycle in master

**Comment by Thrameos** [comment]
This is on hold until I can deal with 3.10 structural issue.   Need to replicate it locally before I can get the release script to work.

**Comment by Thrameos** [comment]
Made some progress on this.  Python 3.10.0b0 is causing serious issues.   It took a while to get pip working at all under it, but even then there are issues.   Currently I am getting segmentation fault during tests that I don't see any reason for.

**Comment by Thrameos** [comment]
Despite fixing the image locally for 3.10, something in the release pipeline is still busted.   Unfortunately this is just holding up everything for something that really shouldn't be built (a distribution for the beta product).   Thus I am just going to remove those from the process and see if I can proceed.

**Comment by nchepanov** [comment]
I'm wondering if uploading to PyPI was going to be part of the process?

**Comment by John-Hennig** [comment]
Was just going to ask the same thing as I'm also waiting for the PyPI release. Thanks for all your efforts, @Thrameos, for getting this one out. (Maybe the PyPI release is something that @marscher would take care of? Pinging just in case.)

**Comment by Thrameos** [comment]
I don't currently hold the keys to PyPI and conda feedstock (though I am sure that marscher would give them if needed).    He usually posts to PyPI and conda up to a week after depending on his time availability.  Most of our release system is automatic through azure pipelines, but the last stage of picking up all the distributions and posting them is manual.   Partly because we don't want to store the keys on azure, but also because it isn't possible to replace a release file on PyPI thus we have to have a human in the loop to validate that azure hasn't posted a bunch of garbage.   Sometimes quality issues are detected in the conda release process that warrant sending it back to the patch release stage as well.   I was worried that may have to happen here as I was getting a local problem which turned out to be version 1.3.0 trying to run with the jar file of 1.2.1.    But that was resolved by cleaning my development system so I believe it should be good to go.

I have not resolved the problems with  manylinux2014_aarch64.   I am not sure why but they posted an image which is running Python 3.10b1 with the flags set for limited API.   That breaks a lot of stuff in JPype because we need some private symbols to implement functions currently.  I really didn't want to stall the release over one bad image especially after I tried for a month to fix Python 3.10 issues only to discover that limited API was also set.

Hopefully the PyPI images will be available soon.
